components/molecules/PasswordResetForm.tsx

```diff
@@ -70,7 +80,23 @@ const PasswordResetForm = () => {
   ]);
 
   const onSubmit = async (data: FormData) => {
-    if (data.confirmNewPassword != data.newPassword) {
+    if (!validPassword(data.newPassword)) {
+      setError(
+        `newPassword`,
+        {
+          message:
+            "The password does not meet the requirements!",
+        },
+        { shouldFocus: true }
+      );
+    }
+    else if (data.currentPassword == data.newPassword) {
+      setError(
+        `confirmNewPassword`,
+        { message: "New password is the same as the current password" },
+        { shouldFocus: true }
+      );
```

Comment:
```suggestion
    else if (data.currentPassword == data.newPassword) {
      setError(
        `newPassword`,
        { message: "New password is the same as the current password" },
        { shouldFocus: true }
      );
```
this error should be on the newPassword field instead of the confirm field.